server/graphql/v2/mutation/AccountingCategoriesMutations.ts

```diff
@@ -68,12 +65,13 @@ export default {
         };
       });
 
+      const newCategories = normalizedInputs.filter(c => !c.id);
       if (
-        isIndependentCollective &&
-        normalizedInputs.some(c => c.appliesTo === AccountingCategoryAppliesTo.HOSTED_COLLECTIVES)
+        !account.hasHosting &&
+        newCategories.some(c => c.appliesTo === AccountingCategoryAppliesTo.HOSTED_COLLECTIVES)
```

Comment:
Changed this one to only enforce it for new categories. If you've used hosting in the past, we don't want to force you to remove the previous accounting categories.